jenkins-design-language/less/components/button.less

```diff
@@ -129,6 +129,10 @@ button, .btn, .btn-default, .btn-primary, .btn-secondary, .btn-danger, a.btn-lin
 button, .btn, .btn-default, .btn-primary, .btn-secondary {
     &.monochrome {
         .btn-styles(transparent, @gray-light, @text-color, @gray-lighter, @gray-lighter);
+
+        svg {
+            fill: @text-color !important;
```

Comment:
I just realised we can fix this easily by setting the fill as an attribute instead of inline style... will try to see if my hunch is correct and if it is... it we can use css AND props to set the colour and hover state